README.md

```diff
@@ -40,58 +40,36 @@ Please remember that this is the latest development build and is **not** tested
 
 ### For Android
 
-#### Quick and dirty
-
 Use the dockerized QField SDK.
 
 ```
-# get the latest tag from https://hub.docker.com/r/opengisch/qfield-sdk/tags/
-TAG=20191001
-cd QField
+QFIELD_SDK=$(awk -F "=" '/osgeo4a_version/{print $2}' sdk.conf)
+ANDROID_NDK_PLATFORM=android-29
 
-git submodule init
-git submodule update
+git submodule update --init --recursive
 
-docker run -v $(pwd):/usr/src/qfield docker.io/opengisch/qfield-sdk:$TAG /usr/src/qfield/scripts/docker-build.sh --rm
+docker run -e ANDROID_NDK_PLATFORM -v $(pwd):/usr/src/qfield docker.io/opengisch/qfield-sdk:$QFIELD_SDK /usr/src/qfield/scripts/docker-build.sh --rm
 ```
 
 This will put the apk into a subfolder `build-docker/out/build/outputs/apk`
 
-#### Go the hard way
-
-Build [OSGeo4A](https://github.com/opengisch/OSGeo4A)
-
-```sh
-cd QField
-
-git submodule init
-git submodule update
-
-cp config.pri.default config.pri
-# Edit config.pri
-```
-
-The advantage of this is, you will be able to build and deploy directly from QtCreator.
-
 ### For Desktop
 
 To build QField for a desktop environment:
 
-* Get QGIS development libraries.
+* Get QGIS development libraries. QField normally uses bleeding edge QGIS code. Ideally install nightly builds or compile it on your own and install to a custom prefix path. See more about QGIS compilation [here](https://github.com/qgis/QGIS/blob/master/INSTALL.md).
+* Download the most recent Qt 5.x version from the [Qt Website](https://www.qt.io/download) to satisfy QField dependencies. Setup a new QtCreator kit that uses the downloaded Qt libs.
```

Comment:
@suricactus two questions:

1. does this work reliably? Don't you need to recompile other dependencies too (like QScintilla, QWT, QtWebKit) in order to get QGIS to compile?
2. on my system I don't need to do this because Qt is new enough, should we flag this instruction as optional?

Reply:
1) for me it was plug-n-play on two Ubuntu machines. At some point I had QField, QGIS and the rest of the Qt dependencies coming compiled with three different versions and it worked. We had this discussion a while ago, as far as  I remember @signedav also confirmed this with Qt 5.13+.
2) sure! I wish I can use my system qt libs for development too. Ubuntu LTS is already lacking 3 versions behind.